.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.1.0
+    hooks:
+      #- id: flake8
+      - id: check-yaml
+        files: .*\.(yaml|yml)$
+  - repo: https://github.com/adrienverge/yamllint.git
+    rev: v1.23.0
+    hooks:
+      - id: yamllint
+        files: \.(yaml|yml)$
+        exclude: ^.travis
```

Comment:
I think travis works fine with formatted yaml files, so I don't think we need an exclude

Reply:
The `.travis.yml` has some lines too long. I've adapted the configuration of `yamlint` to relax this constraint.